QATCH/VisQAI/src/view/table_view.py

```
@@ -1,237 +1,535 @@
+            raw_item (Any): The data object for the cell (str, int, float, list, or dict).
+        """
+        # Determine if this is a structural column
+        is_label_col = col == 0
+        is_unit_col = col == 2
+
+        # Handle Structural Columns or Simple Strings
+        if (
+            is_label_col
+            or is_unit_col
+            or isinstance(raw_item, (str, int, float))
+            and not isinstance(raw_item, (dict, list))
```

Ruff (0.14.14)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)